src/main/java/au/org/ala/names/search/ALANameSearcher.java

```diff
@@ -1007,16 +1009,17 @@ public String searchForLsidById(String id) {
             throw new SPPException();//SearchResultException("Unable to perform search. Can not match to a subset of species within a genus.");
 
         try {
+            CleanedScientificName cleaned = new CleanedScientificName(name);
```

Comment:
The CleanedScientificName looks useful. Further work might be to feed this back into GBIF's name parser code.